src/translation/translations/en.json

```
@@ -532,11 +534,15 @@
     "governanceProposals": "Governance proposals",
     "pasteYourAddress": "Paste your address",
     "pleasePayAttention": "Please pay attention",
+    "redelegate": "Redelegate",
+    "redelegateVoting": "Redelegate voting",
+    "requiredVotingPower": "You need to have 300k voting power to create proposal",
```

I think "a" is missing in that sentence, right?
```suggestion
    "requiredVotingPower": "You need to have 300k voting power to create a proposal",
```

---

Got you!